### ISSUE TITLE
Forgotten Translations

### ISSUE THREAD

**Issue by pergolafabio** [MortenVinding/Robomow-ESP32#13]
Hi,  gonna paste some forgotten translations here:

Next operation :	Wednesda 14:18 **Uhr**
Operation obstruct:	Mähzeit abgeschlossen

**Comment by motogon** [comment]
I also noticed in German : 
Bat. lädt/balanciert
Bat. lädt
46 Überhitzung Mow-motor - Erholungsversuch (46 	Mow overheat 	Mow overheat is detected during automatic operation but operation is not yet stopped and robot is trying to recover from it.)

Konto Schlüssel ( Key to google maps api  )

**Comment by MortenVinding** [comment]
> I also noticed in German

Great thank you,
I have tried to translate that as good as I could also (being a binary file, it has to be translated to same length!).

**Comment by pergolafabio** [comment]
Operation obstruct:	Mäher im On satz oder nur not in Ladestation

**Comment by pergolafabio** [comment]
hey @MortenVinding , are you still working on this one to change translations?

**Comment by pergolafabio** [comment]
especially the status events, like below :-)

![Image](https://github.com/user-attachments/assets/2917e7e4-72b4-4510-807a-23468e5722a7)

**Comment by MortenVinding** [comment]
Sorry I have not putten much love in to this recently.
Hmm I can't find any references to that in the latest .bin file, are you sure you are using the latest one?

**Comment by pergolafabio** [comment]
Yeah probably, I don't think you did changes to bin after June 2024,? :-)
